Bastion/src/isaac/bastion/listeners/EnderPearlListener.java

```diff
@@ -1,42 +0,0 @@
-package isaac.bastion.listeners;
-
-import isaac.bastion.Bastion;
-import isaac.bastion.manager.BastionBlockManager;
-import isaac.bastion.manager.EnderPearlManager;
-
-import org.bukkit.Server;
-import org.bukkit.entity.EnderPearl;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.EventPriority;
-import org.bukkit.event.Listener;
-import org.bukkit.event.entity.ProjectileLaunchEvent;
-import org.bukkit.event.player.PlayerTeleportEvent;
-
-public class EnderPearlListener implements Listener {
```

Comment:
There were only two methods here so I moved them to BastionDamageListener since they both fit in that category, could move them back out to a separate class though